tests/testthat/test-use-circleci.R

```diff
@@ -0,0 +1,38 @@
+context("use_circleci")
+
+test_that("uses_circleci() reports usage of CircleCI", {
+  scoped_temporary_package()
+  expect_false(uses_circleci())
+  use_git()
+  git2r::remote_add(name = "origin", url = "https://github.com/fake/fake")
```

Comment:
Please switch over to `usethis::use_git_remote()`. We will probably switch over to gert at some point, so I don't want to acquire any new git2r calls.